<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyRobotActionForm</class>
 <widget class="GUIFormBase" name="PropertyRobotActionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButtonMove">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>79</height>
     </rect>
    </property>
    <property name="text">
     <string>移動</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonCapture">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>121</width>
      <height>79</height>
     </rect>
    </property>
    <property name="text">
     <string>撮影・検査</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>動作追加</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetMarkList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>441</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>動作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>タイプ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>検査</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="toolButtonTeachingMode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ティーチングモード</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClearAll">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>全グリア</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetRobotList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
